init.go

```diff
@@ -126,6 +126,15 @@ func (a *initFnCmd) init(c *cli.Context) error {
 		}
 	}
 
+	const runHeader = `
```

Comment:
Seems a bit much for a command like this... 🤷‍♀️   Lots of extra lines with no real value.

Reply:
it's either there or when you install the CLI... Serverless does it at their init command. I think it fits. People in theory won't be fn init'ing all the time.

Reply:
Ping. Value is branding and small touches. Can we merge?